.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+---
+version: 2
+updates:
+
+  # GitHub actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+      time: "22:00"
```

Comment:
**@williammartin : how consistently should Go and GitHub Actions dependabot PRs be across the `cli` repositories?  how frequently do you expect to address updates?**

I ask because [`cli/cli` dependabot settings](https://github.com/cli/cli/blob/31a9d083bd0feaf237e7cf33b6171a1e902b7d3a/.github/dependabot.yml#L12-L15) differ quite a bit and appreciate consistency:

```yaml
version: 2
updates:
- package-ecosystem: gomod
  directory: "/"
  schedule:
    interval: "daily"
  ignore:
    - dependency-name: "*"
      update-types:
        - version-update:semver-minor
        - version-update:semver-major
- package-ecosystem: "github-actions"
  directory: "/"
  schedule:
      interval: "daily"
```

Between these changes and `cli/cli`, I think the `groups` usage could help cut down on churn of multiple PRs.